app/scripts/modules/amazon/instance/details/instanceDetails.html

```diff
@@ -120,44 +120,44 @@ <h3 select-on-dbl-click>
           <a href="http://{{instance.privateDnsName}}:{{state.instancePort}}" target="_blank">{{instance.privateDnsName}}</a>
           <copy-to-clipboard
               class="copy-to-clipboard copy-to-clipboard-sm"
-              text="{{instance.privateDnsName}}"
-              tool-tip="Copy to clipboard">
+              text="instance.privateDnsName"
+              tool-tip="'Copy to clipboard'">
```

Comment:
This whole pattern is because copy-to-clipboard was switched to use one-way bindings